challenges/towers-of-hanoi/towers_of_hanoi.py

```diff
@@ -0,0 +1,13 @@
+def towers_of_hanoi(n):
+    """Validate tower argument and trigger recursion."""
+    if type(n) is not int:
+        raise TypeError('Please enter an integer!')
+    return(move_tower(n, 'A', 'C', 'B'))
+
+
+def move_tower(discs, origin, target, helper):
+    """Handle recursive shuffle of discs."""
+    if discs > 0:
+        move_tower(discs - 1, origin, helper, target)
+        return(f'move disk from {origin} to {target}')
```

Comment:
This `return` statement is stopping your recursion early so that the solution generated is incomplete and incorrect.